addons/c130/config.cpp

```diff
@@ -0,0 +1,92 @@
+class CfgPatches {
+
+    class RHS_US_A2_AirImport {
+        units[] = {};
+        weapons[] = {};
+        requiredVersion = 1.320000;
+        requiredAddons[] = {"A3_Air_F", "A3_Air_F_Beta", "A3_Air_F_EPC_Plane_CAS_01", "A3_Air_F_EPB_Heli_Light_03", "A3_CargoPoses_F", "rhsusf_c_troops", "rhsusf_c_heavyweapons", "rhsusf_sounds", "rhsusf_c_airweapons"};
+        name = "RHSUSF c130 sound range fix";
+        author = "nomisum";
+    };
+};
+class CfgVehicles {
+    /*extern*/ class Air;
+
+    class Plane: Air {
+        /*extern*/ class HitPoints;
+    };
+
+    class Plane_Base_F: Plane {
+        /*extern*/ class NewTurret;
+        /*extern*/ class Turrets;
+        /*extern*/ class ViewPilot;
+        /*extern*/ class ViewGunner;
+        /*extern*/ class AnimationSources;
+
+        class HitPoints: HitPoints {
+            /*extern*/ class HitHull;
+        };
+    };
+
+    class Plane_CAS_01_base_F: Plane_Base_F {
+        /*extern*/ class EventHandlers;
+    };
+
+    class RHS_C130J_Base: Plane_Base_F {
+    };
```

Comment:
This could be a lot simpler 😅 and would also reduce the number of `requiredAddons`.

```suggestion
    class RHS_C130J_Base;
```